packages/shared/src/utilities/react/updateArray.ts

```diff
@@ -0,0 +1,13 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
```

Comment:
Could you add a JSDoc comment here to explain what this utility actually does? Maybe also rename `updateArray` to something more descriptive.

Reply:
Added the comment and renamed